migration/migration_blackbox_test.go

```diff
@@ -335,6 +336,25 @@ func testMigration61(t *testing.T) {
 
 }
 
+func testMigration62(t *testing.T) {
+	migrateToVersion(sqlDB, migrations[:(initialMigratedVersion+18)], (initialMigratedVersion + 18))
+
+	assert.True(t, gormDB.HasTable("categories"))
+	assert.True(t, dialect.HasIndex("categories", "categories_id_index"))
+	assert.True(t, dialect.HasIndex("categories", "categories_name_idx"))
+
+	assert.True(t, gormDB.HasTable("work_item_type_categories"))
+	assert.True(t, dialect.HasIndex("work_item_type_categories", "work_item_type_categories_id_idx"))
+	assert.True(t, dialect.HasIndex("work_item_type_categories", "work_item_type_categories_idx"))
+
+	assert.True(t, dialect.HasColumn("work_item_type_categories", "category_id"))
+	assert.True(t, dialect.HasColumn("work_item_type_categories", "work_item_type_id"))
+
+	// These script execution has to fail
+	assert.NotNil(t, runSQLscript(sqlDB, "062-insert-category-no-name-fail.sql"))
+	assert.NotNil(t, runSQLscript(sqlDB, "063-unique-idx-failed-insert-category.sql"))
```

Comment:
@kwk Please ellaborate this -> 

> Why not insert a few categories? This will aid the migration later.

I already insert two categories during migration (https://github.com/almighty/almighty-core/pull/1037/files/#diff-194905cabff622d515b0957a60cba773R23)